### PR TITLE
001 test modify

### PR DESCRIPTION
**Pull request by isidro1984**
Hi Goran.

I send you a pull request of my respository with the first issue.  I added up all files for merge with your repository. 

I await your confirmation.

### CODE REVIEW

File: www/index.php [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <?php
 include "../core/basics.php";
 
-if (file_exists(PATH_SERVICES . Router::$service . ".php")) {
-    CORE::$SERVICE = new Service(Router::$service);
+if (file_exists(PATH_SERVICES . Router::$params[1] . ".php")) {
```
Comment (schkovich) [inline www/index.php]: This does not work for me at all. Router::$params[1] is null. I am getting 404 always.

File: services/get.php [hunk 2]
```diff
@@ -28,30 +52,131 @@ function loadFeed($feedUrl)
...
+        echo IO::error($e->getMessage());
+    }
 }
+
+//This function removes any vacancy which no is identical to category
+//@input all vacancies and category to match
+//@output vacancies with the matched category
+function vacanciesByCategory($vacancies, $category)
+{
+    $category = str_replace("%20", " ", $category);
```
Comment (isidro1984) [inline services/get.php]: When the parameter "category" is introduced by URL, if this category has "blank" spaces, code didn't recognize it well. For this reason I had to use this function to change "%20" to  "   ".

File: www/index.php (hunk 1 above)
Comment (isidro1984) [inline www/index.php]: image

Original index.php code didn't work for me except if It was set up in the root of my XAMPP. If you want to set up in another folder inside It doesn't work. As I had in my htdocs folder another projects I had to solve in another way. For this reason I changed the Index.php to get the first paramter by url. Changing the .htaccess as I explain in the manual it works in other directories of the server.

Comment (schkovich) [inline www/index.php]: hmhh... it does not make sense to solve your problem and create problem for others. you might consider finding a better solution for your development environment.

i tried changing root to `/srv/tom/apitest` but it did not help.

image

any suggestions how to make it work? i'm running nginx and php-fpm.

Comment (isidro1984) [inline www/index.php]: Have you changed the .htaccess? 

image

image

Comment (schkovich) [inline www/index.php]: no, i did not. do you consider that it will make a difference?

Comment (isidro1984) [inline www/index.php]: You are right. I have tested this index.php in another empty environment and it doesn't work unless you set up it in an inside folder changing the .htaccess too. It gives an error with the first parameter (undefined index). $params['0'] it was a bad idea for do this thing. I always will receive an "undefinded index notice". ". 

image

image

**Pull request closed without merging**